blendergltf.py

```diff
@@ -482,22 +451,34 @@ def export_mesh(me):
             # Find the (vertex) index associated with each loop in the polygon.
             indices = [vert_dict[i].index for i in poly.loop_indices]
 
-            # Record the maximum index. This isn't used now but may be used in
-            # the future to determine whether a mesh needs to be split or if
-            # integer indices are to be used, etc.
+            # Used to determine whether a mesh must be split.
             for i in indices:
```

Comment:
Using the built in max function might be faster here:
https://docs.python.org/3.5/library/functions.html#max